pkg/asset/rhcos/image.go

```diff
@@ -86,8 +85,10 @@ func osImage(config *types.InstallConfig) (string, error) {
 			return config.Platform.AWS.AMIID, nil
 		}
 		region := config.Platform.AWS.Region
-		if !configaws.IsKnownRegion(config.Platform.AWS.Region, config.ControlPlane.Architecture) {
-			region = "us-east-1"
+		if _, ok := streamArch.Images.Aws.Regions[region]; !ok {
+			globalResourceRegion := "us-east-1"
```

Comment:
open to suggestions here for a better variable name

Reply:
amiSourceRegion?

Reply:
Also, nit that this can be a constant.
```suggestion
			const globalResourceRegion = "us-east-1"
```